src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+For user experience, we want LCP elements to load as fast as possible. User experience is why LCP was selected as one of the Core Web Vitals. We do not want it to be lazy-loaded nor decoded asynchronously as that further delays the render. However, we can see that 9.3% of pages used the native loading=lazy flag on the LCP `<img>` element.
+
+Not all browsers support native lazy loading. Popular lazy loading polyfills detect a "lazyload" class on an image element. Thus, we can identify more possibly lazy-loaded images by adding images with a "lazyload" class to the total. The percent of sites probably lazy loading their LCP `<img>` element jumps up to 16.5% on mobile.
+
+Lazy loading your LCP element will result in worse performance. *Don't do it!* WordPress was an early adopter of native lazy loading. The early method was a naive solution applying lazy loading to all images, and the results showed a <a hreflang="en" href="https://web.dev/lcp-lazy-loading/">negative performance correlation</a>. They were able to use this data to implement a more nuanced approach for better performance.
+
+Luckily, only 0.4% of sites used the async decode directive for their LCP image. The negative impact of asynchronous decode is not as high as lazy loading.
+
+{{ figure_markup(
+  content="354",
+  caption="Websites attempted to use native lazy-loading on LCP elements that are not images or iframes",
+  classes="big-number",
+  sheets_gid="1423728540",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+Interestingly, 354 origins on desktop attempted to use native lazy-loading on HTML elements that do not support the loading attribute (e.g., `<div>`). The loading attribute is only supported on `<img>` and  sometimes `<iframe>` elements (see <a hreflang="en" href="https://caniuse.com/loading-lazy-attr">caniuse</a>).
+
+### Cumulative Layout Shift (CLS)
+
+{{ figure_markup(
+  image="performance-CLS-by-device.png",
+  caption="CLS performance by device",
+  description="Stacked bar chart showing CLS performance for desktop and mobile devices. For desktop devices, 62% of websites had good (< 0.1), 23% needs improvement, and 15% poor (>= 0.25) CLS performance. For mobile, 62% were good, 21% needs improvement, and 17% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=160840238&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+<a hreflang="en" href="https://web.dev/cls/">Cumulative Layout Shift (CLS)</a> is characterized by how much shift a user experiences, not how long it takes to visually see something like FCP and LCP. As such, performance by device was fairly equivalent.
```

Comment:
"how much shift a user experiences" is a little non-descriptive.

```suggestion
<a hreflang="en" href="https://web.dev/cls/">Cumulative Layout Shift (CLS)</a> is characterized by how much content moves around or "shifts" while viewing a web page, not how long it takes to visually see something like FCP and LCP. As such, performance by device was fairly equivalent.
```

Reply:
That seems long winded and more difficult to read, so I might change it to something else. Make sure you run complex sentences through Hemingway.app